setup.py

```diff
@@ -81,7 +81,7 @@ def run(self):
         "netcdf4",
         "pyyaml",
         "pyshp",
-        "scipy",
+        "scipy<1.3",
         "six",
```

Comment:
@mcflugen : Can you turn your comment to @saisiddu and @RondaStrauch into an issue so we can keep track of this beyond closing this PR?

Reply:
Definitely. I intended to do just that.

Reply:
Done.